src/main/java/com/frcteam3636/frc2024/subsystems/intake/Intake.kt

```diff
@@ -20,14 +20,21 @@ object Intake : Subsystem {
     }
 
     fun intakeCommand(): Command {
-        return InstantCommand({
-            io.setUnderBumperRoller(1.0)
-            io.setOverBumperRoller(1.0)
-        })
-            .until(io::isIntaking)
+        return StartEndCommand(
```

Comment:
I think this could use a `Subsystem` command factory so we don't have to add ourselves as a requirement manually.